buildozer/__init__.py

```diff
@@ -879,10 +871,27 @@ def namify(self, name):
     def root_dir(self):
         return realpath(dirname(self.specfilename))
 
+    @property
+    def user_build_dir(self):
+        '''The user-provided build dir, if any.'''
+        # Check for a user-provided build dir
+        # Check the (deprecated) builddir token, for backwards compatibility
+        build_dir = self.config.getdefault('buildozer', 'builddir', None)
+        if build_dir:
+            # for backwards compatibility, append .buildozer to builddir
+            build_dir = join(build_dir, '.buildozer')
+        build_dir = self.config.getdefault('buildozer', 'build_dir', build_dir)
+
+        if build_dir is not None:
```

Comment:
nitpick: this is Pythonic, but inconsistent with the non-pythonic way of checking just be fore in line 880 `if build_dir:`